include/aws/io/retry_strategy.h

```diff
@@ -0,0 +1,175 @@
+#ifndef AWS_IO_CLIENT_RETRY_STRATEGY_H
+#define AWS_IO_CLIENT_RETRY_STRATEGY_H
+/*
+ * Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+#include <aws/io/event_loop.h>
```

Comment:
trivial: I think you could get away with fwd declares here

Reply:
yes, you're right, will fix.